### ISSUE TITLE
Skydive from owl using default Sprite instead of custom sprite

### ISSUE THREAD

**Issue by busybody42** [SuperTux/supertux#789]
**Supertux version 0.5.1:**
**System information:** *64 bit Windows 10*

##### Additional debugging information

*If available, please attach a backtrace of a crash (you can get one using GDB or another debugger), or a core dump. If you're on GNU/Linux and your distro is using systemd, it's highly likely that a core dump is available using `coredumpctl(1)`.
If the issue only occurs in a custom level, please attach that too.*

##### Guidelines for reporting issues

- [ ] I have read <https://github.com/SuperTux/supertux/blob/master/CONTRIBUTING.md#bug-reports>.
- [ ] I verified this isn't an issue that's already been reported.
- [ ] This report only includes details about **one** (1) issue, feature request or topic.
- [ ] If I make a mistake while submitting this report I will use the edit feature instead of closing this issue and opening a new one.
- [ ] This issue is **not** about wrong translations (use Transifex for those), or anything unsupported (e.g. third-party add-ons).
- [ ] Once you're done, please remove the section 'Guidelines for reporting issues', the 'Additional debugging information' if it is empty, and any text in *italic font*.

**Comment by tobbi** [comment]
Example level, please.

Also, @brmbrmcar Tell @busybody42 what happens if they keeps abusing github reactions.

**Comment by brmbrmcar** [comment]
You get blocked.

**Comment by busybody42** [comment]
(I don't know how to get the code for custom levels) BTW, you don't NEED an example level for this! It can happen in ANY level! (btw, sorry about that, I didn't know about the reactions!!)

**Comment by maxteufel** [comment]
Yes, you need an example level. Not all levels include skydives and owls, and not all levels that use those badguys use custom sprites.

**Comment by busybody42** [comment]
There is one in my custom level, "A 'shocking' Experience" that uses a custom sprite! When I find out how to get the code for custom levels, I will post it!

**Comment by busybody42** [comment]
[level3.zip](https://github.com/SuperTux/supertux/files/1839462/level3.zip)
The level 'A "shocking" experience' the cave bit has a owl with skydive, that both use custom sprites.

**Comment by busybody42** [comment]
I just noticed that when a skydive should use a custom sprite, it spawns another one that DOES use the sprite, instead of just having a skydive with a custom sprite attached to the owl. :/

**Comment by busybody42** [comment]
Basically, a skydive using the default Sprite spawns for the owl, and the one in the editor is NOT attached to the owl, but it automatically spawns when you place down an owl, and THAT skydive is the one that uses a custom Sprite :/

**Comment by busybody42** [comment]
The owl spawns it's own skydive in play mode, but for some reason, it also places a random skydive as well!

**Comment by tobbi** [comment]
It seems like this issue is a duplicate of #718. You cannot currrently change the sprite of the owl's carried object using the editor. And given how much people abuse the ability, I don't see a reason for it being any other way.

**Comment by busybody42** [comment]
Oh, sorry. I didn't know that issue was already reported. (by the way, i was trying to make it look like there were flying snowballs carrying bombs for custom levels, but this issue got in the way)

**Comment by tobbi** [comment]
Guys, should we allow changing the sprite of an owl's payload?
